dashboard/modules/reporter/reporter_agent.py

```diff
@@ -779,6 +890,15 @@ def _record_stats(self, stats, cluster_stats):
                     tags={"ip": ip},
                 )
             )
+
+            if total_workers_shm > 0.0:
```

Comment:
Tracking max worker memory usage and worker id somewhere might be helpful. Wondering if we could do that in the future. 

Thoughts on per worker usage metric?  It could be gated if the cardinality overhead is too big.